pom.xml

```diff
@@ -1095,6 +1095,7 @@
 
             <!-- Used in internal utilities (protected by conditional) -->
             <ignore>java.lang.Long</ignore>
+            <ignore>java.lang.Character</ignore>
```

Comment:
@normanmaurer - This is not ideal. These two should only be exposed in the "common" sub module. On top of that there are only 2 static methods which are used. I spent a small amount of time on this but just gave in an added it to the root pom...